content/docs/1.31/operator.md

```diff
@@ -23,6 +23,14 @@ While multiple operators might coexist watching the same set of namespaces, whic
 The Jaeger Operator version tracks one version of the Jaeger components (Query, Collector, Agent). When a new version of the Jaeger components is released, a new version of the operator will be released that understands how running instances of the previous version can be upgraded to the new version.
 {{< /info >}}
 
+## Prerequisite
+
+Since version 1.31 the Jaeger Operator uses webhooks to validate Jaeger custom resources (CRs). This requires an installed version of the [cert-manager](https://cert-manager.io/docs/). A more detailed list of supported versions can be found in the [compability matrix](https://github.com/jaegertracing/jaeger-operator#compatibility-matrix). An installation guide can be found [here](https://cert-manager.io/v1.6-docs/installation/#default-static-install).
```

Comment:
> uses webhooks

what is the request flow here? I don't understand this part of the operator.

Reply:
also, can this validation be turned off?

Reply:
> what is the request flow here? I don't understand this part of the operator.

![image](https://user-images.githubusercontent.com/10403402/158679479-2f20abcb-dc24-4d2b-a866-24d8f91dcb6c.png)
when a Jaeger CR is created, the validation step is used to check if Elasticsearch has been configured and an instance exists.

> also, can this validation be turned off?

Not so far. But probably version 1.33 and later we will use mutating webhooks for the pod based sidecar injection. Means further usecases will depend on it.

Reply:
Would it help like this?

```suggestion
Since version 1.31 the Jaeger Operator uses webhooks to validate Jaeger custom resources (CRs) when they are created or updated. This requires an installed version of the [cert-manager](https://cert-manager.io/docs/). A more detailed list of supported versions can be found in the [compability matrix](https://github.com/jaegertracing/jaeger-operator#compatibility-matrix). An installation guide can be found [here](https://cert-manager.io/v1.6-docs/installation/#default-static-install).
```